plugins/dataset_feature_space.cc

```diff
@@ -114,6 +114,9 @@ getColumnInfo(std::shared_ptr<Dataset> dataset,
                 auto categorical = std::make_shared<ML::Fixed_Categorical_Info>(allValues);
                 result.info = ML::Feature_Info(categorical);
             }
+            else if (types.size() == 1 && types.begin()->first == CellValue::EMPTY) {
```

Comment:
Not sure I understand your comment.  I mean this if case is taken, we should set the `result.info` to `ML::UNKNOWN` and not `ML::INUTILE`.